pkg/ui/ccl/src/views/reports/containers/stores/encryption.tsx

```diff
@@ -14,16 +28,77 @@ export default class EncryptionStatus extends React.Component<EncryptionStatusPr
     );
   }
 
-  render(): React.ReactElement<any> {
+  renderKey(isStoreKey: boolean, key: protos.cockroach.ccl.storageccl.engineccl.enginepbccl.KeyInfo$Properties) {
```

Comment:
please no mystery boolean parameters.

at a first glance the two cases seem to share almost nothing.  Looking closer, they share an awful lot that's just been copy-pasted.  please clean this up

Reply:
Split into separate functions.